codespell_lib/data/dictionary.txt

```diff
@@ -30614,6 +30620,7 @@ thess->this, these,
 thest->test
 thether->tether, whether,
 thetrahedron->tetrahedron
+thets->that's
```

Comment:
Suggest theta too

Reply:
Just replaced with `thets->that's, theta,`